doc/Command-Reference.md

```diff
@@ -1742,11 +1742,17 @@ This command displays some more fields such as Lanes, Speed, MTU, Type, Asymmetr
   show interface status of all interfaces
   
   admin@sonic:~$ show interfaces status
-  Interface            Lanes    Speed    MTU            Alias    Oper    Admin    Type    Asym PFC
-  -----------  ---------------  -------  -----  ---------------  ------  -------  ------  ----------
-  Ethernet0      49,50,51,52     100G   9100   hundredGigE1/1    down       up     N/A         off
-  Ethernet4      53,54,55,56     100G   9100   hundredGigE1/2    down       up     N/A         off
-  Ethernet8      57,58,59,60     100G   9100   hundredGigE1/3    down     down     N/A         off
+    Interface            Lanes    Speed    MTU           Alias           Vlan    Oper    Admin    Type    Asym PFC    Neighbour         Port

```

Comment:
This output data is not displaying properly in the rendered Markdown. Please fix.

Reply:
corrected in next iteration

Reply:
corrected in next iteration